src/middleware/finalErrorHandler.js

```
@@ -0,0 +1,6 @@
+const { messages: errMessages } = require('../constants/errors')
+
+module.exports = (err, req, res) => {
+  console.error(err)
+  res.boom.badImplementation(errMessages.UNHANDLED_CUSTOM)
```

I think Boom eats this error message. So you can leave the argument blank!